ui/app/styles/core/bulma-classes.css

```diff
@@ -252,81 +252,3 @@ fieldset[disabled] .radio,
   background-color: #48c78e;
   color: #fff;
 }
-
```

Comment:
Was able to remove allll of this css by not having the one-off `delete` class on the modal.

Reply:
Crazy how many classes there are for delete!